bin/hdfcoinc/pycbc_combine_coincident_events

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""
+This program combines together a set of STATMAP files from disjoint times.
+The resulting file would contain triggers from the full set of input files
+"""
+import numpy
+import argparse
+import logging
+import h5py
+
+import lal
+
+import pycbc
+import pycbc.events
+import pycbc.version
+
+import h5py, numpy, argparse, logging, pycbc, pycbc.events, lal
+import pycbc.version
+
+def com(f, files, group):
+    """ Combine the same column from multiple files into another file f"""
+    try:
+        f[group] = numpy.concatenate([fi[group][:] if group in fi else numpy.array([], dtype=numpy.uint32) for fi in files])
+    except:
+        print group
+        raise
+
+def com_with_swapping(f, files, group):
+    """
+    Combine data from multiple files where the group is dependent on which of
+    the two ifos the data belongs to, e.g. time1, time2 or trigger_id1,
+    trigger_id2. This function checks which of the two groups should be used
+    for each file, as each file might have a different convention, and combines
+    according to the attributes set in f.
+    """
+    # Set what the group should be for the "other" ifo
```

Comment:
Is there a way to make this Virgo-proof, i.e. work with more than 2 ifos involved?  it should be possible to positively identify which group corresponds to a specific detector name.

Reply:
Discussed below